pkgs/applications/networking/browsers/chromium/update.py

```diff
@@ -90,28 +90,32 @@ def get_channel_dependencies(version):
     }
 
 
-def get_latest_ungoogled_chromium_tag():
-    """Returns the latest ungoogled-chromium tag using the GitHub API."""
-    api_tag_url = 'https://api.github.com/repos/Eloston/ungoogled-chromium/tags?per_page=1'
+def get_latest_ungoogled_chromium_tag(linux_stable_versions):
+    """Returns the latest ungoogled-chromium tag for linux using the GitHub API."""
+    api_tag_url = 'https://api.github.com/repos/ungoogled-software/ungoogled-chromium/tags'
```

Comment:
Does this take a github token?

Reply:
I mean technically yes but I would assume that the package maintainers don't run the script to an extent that would require a token

Reply:
The rate limiting is per IP for the entire API. If you are in a shared space then the rate limiting is often very quickly hit.

Reply:
So you want an optional flag that passes a github api token to the program?

Reply:
It would be great if it just takes one from GITHUB_TOKEN env.